core/client/pom.xml

```diff
@@ -256,7 +221,7 @@
               <relocations>
                 <relocation>
                   <pattern>org.apache.thrift</pattern>
-                  <shadedPattern>shaded.alluxio.org.apache.thrift</shadedPattern>
+                  <shadedPattern>${project.artifactId}.org.apache.thrift</shadedPattern>
```

Comment:
I am not sure what benefit would that provide ... whatever value we use for `${project.shading.prefix}` could be just inlined here, no?